src/main.jsx

```diff
@@ -5,11 +5,18 @@ import { RouterProvider, createBrowserRouter } from "react-router-dom";
 import App from "./App";
 
 import "./index.css";
+import ModalWrapper from "./components/Modal/ModalWrapper";
```

Comment:
```suggestion
import App from "./App";
import ModalWrapper from "./components/Modal/ModalWrapper";

import "./index.css";
```

import 구문이 종류별로 구분되어 있으면 좋을 것 같습니다.

Reply:
라이브러리

컴포넌트

기타 파일

순으로 수정했습니다